src/pages/leaderboard.js

```diff
@@ -89,7 +89,7 @@ const Leaderboard = ({ data }) => {
   const handles = data.handles.edges;
 
   const resultData = useMemo(() => {
-    const result = [];
+    let result = [];
```

Comment:
Fair enough, but re-writing the return statement might be better in this case.

```suggestion
    const result = [];
```